pymatgen/io/vasp/outputs.py

```diff
@@ -766,33 +786,39 @@ def run_type(self) -> str:
         if self.parameters.get("LUSE_VDW", False):
             run_type += "+rVV10"
         elif self.incar.get("IVDW") in IVDW_TYPES:
-            run_type += "+vdW-" + IVDW_TYPES[self.incar.get("IVDW")]
+            run_type += f"+vdW-{IVDW_TYPES[self.incar.get("IVDW", 0)]}"
```

Comment:
No sure if we really need the default `IVDW` as `0` (for no correction), see [VASP wiki](https://www.vasp.at/wiki/index.php/IVDW)

Reply:
Please be cognizant of downstream effects, especially in the materials project software stack. If there is no reason to remove something, then it may be best to leave as-is in case something downstream depends on it

Reply:
I'm not quite sure about downstream effect, but I assume default of `0` instead of `None` might align better with VASP behavior. I would be happy to revert this though, if you decide it might bring more risk than benefits :)

Reply:
Perhaps we need to revisit this change? 

With current implementation, if the `IVDW` tag is absent from `INCAR`, `self.incar.get('IVDW')` would return `None`, and `f"+vdW-{IVDW_TYPES[self.incar.get('IVDW')]}"` would break by using `None` as the index.

Also I think setting the default value of `IVDW` is a rational default which agrees with the [VASP wiki](https://www.vasp.at/wiki/index.php/IVDW):
>IVDW = 0 | 1 | 10 | 11 | 12 | 13 | 2 | 20 | 21 | 202 | 263 | 3 | 4
Default: IVDW = 0 (no correction) 
Description: IVDW specifies a vdW (dispersion) correction.